api/lib/domain/errors.js

```diff
@@ -510,6 +505,14 @@ class CertificationCandidateByPersonalInfoNotFoundError extends DomainError {
   }
 }
 
+class CertificationCandidateNotFoundError extends DomainError {
+  constructor(message = 'No candidate found') {
+    super(message);
+    this.code = 'CANDIDATE_NOT_FOUND';
+    this.detail = 'No candidate found';
+  }
+}
```

Comment:
suite au retour d'avant

```suggestion
class CertificationCandidateNotFoundError extends DomainError {
  constructor(message = 'No candidate found') {
    super(message);
    this.code = 'CANDIDATE_NOT_FOUND';
  }
}
```